ci/pipeline.yml

```diff
@@ -0,0 +1,492 @@
+---
+############################
+#  SHARED
+
+staging-cf: &staging-cf
+  CF_API: https://api.fr.cloud.gov
+  CF_USERNAME: ((cf-username))
+  CF_PASSWORD: ((cf-password))
+  CF_ORG: gsa-18f-federalist
+  CF_SPACE: staging
+
+node-image: &node-image
+  type: docker-image
+  source:
+    repository: node
+    tag: 14
+
+cf-image: &cf-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: 18fgsa/concourse-task
+
+test-api: &test-api
+  - in_parallel:
+    - get: redis
+      params: {save: true}
+    - get: postgres
+      params: {save: true}
+    - get: node
+      params: {save: true}
+
+  - task: install-deps-api
+    config:
+      platform: linux
+      image_resource:
+        <<: *node-image
+      inputs: [name: src-api]
+      outputs: [name: src-api]
+      run:
+        dir: src-api
+        path: bash
+        args: [-c, yarn --pure-lockfile]
+
+  - in_parallel:
+    - task: lint-api
+      config:
+        platform: linux
+        image_resource:
+          <<: *node-image
+        inputs: [name: src-api]
+        run:
+          dir: src-api
+          path: bash
+          args: [-c, yarn lint]
+
+    - task: test-api
+      privileged: true
+      config:
```

Comment:
Maybe worth splitting these larger tasks into separate files later on to get the pipeline file a bit smaller.

Reply:
yes, there is a LOT of refactoring that can happen here...